src/Console/Commands/CheckCommand.php

```diff
@@ -30,8 +32,7 @@ final class CheckCommand extends Command
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $configurationPath = $input->getOption('config');
-        Config::resolveConfigFilePathUsing(fn (): mixed => $configurationPath);
+        $this->getConfigFile($input);
```

Comment:
```suggestion
        $this->intializeConfigFromParameter($input);
```

Reply:
I added this suggestion (and the following) to add a more concise name to the function. With `get` you would expect the function to return something. This initializes the Config.